browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -3618,9 +3616,6 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_BRAVE_WALLET_SEARCHING_FOR_DOMAIN" desc="Searching for wallet domain progress message.">
       Searching for domain...
     </message>
-    <message name="IDS_REWARDS_TIPPING_SELECT_AMOUNT" desc="Select tipping amount text">
-      Please select amount to tip
-    </message>
     <message name="IDS_BRAVE_WALLET_NFTS" desc="NFTs as title">
       NFTs
     </message>
```

Comment:
in general description is something that helps people translate more accurate. I don't think a description of  for example `Send amount` string should be `Text for Send amount`. It should briefly describe a context of what that string represents.